docker-compose.yml

```diff
@@ -109,10 +109,10 @@ services:
   flextesa:
     container_name: deku_flextesa
     restart: always
-    image: oxheadalpha/flextesa:20220127
-    command: hangzbox start
+    image: oxheadalpha/flextesa:20220510
+    command: jakartabox start
     environment:
-      - block_time=4
+      - block_time=30
```

Comment:
This slows down the local testing significantly, especially if we pair it with a more realistic `--unsafe_tezos_required_confirmations` in `sandbox.sh`, might be good for the devs to test with this, especially when  doing realistic benchmarks.

Reply:
Moved this change and added the sandbox.sh into a separate commit that we can drop before merging this, just want to showcase a more realistic scenario